spec/flipper/ui_spec.rb

```diff
@@ -146,4 +146,15 @@
       described_class.feature_creation_enabled = @original_feature_creation_enabled
     end
   end
+
+  context 'with RACK_ENV set to production' do
+    before do
+      ENV["RACK_ENV"] = "production"
```

Comment:
since `ENV` is global this will pollute the rest of the tests with `ENV["RACK_ENV"] = "production"`.  One solution would be to stub it.